src/byte/views/abstract_views.py

```diff
@@ -0,0 +1,88 @@
+"""Inheritable views that include extra functionality for base Views classes."""
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from discord import ButtonStyle, Embed, Interaction
+from discord.ui import Button, View, button
+
+if TYPE_CHECKING:
+    from typing import Self
+
+    from discord.ext.commands import Bot
+
+__all__ = ("BaseEmbedView",)
+
+
+class BaseEmbedView(View):
```

Comment:
I like this idea - but in the other views its basically small/pointless? Idk whats better